### ISSUE TITLE
Label landmark elements properly

### ISSUE THREAD

**Issue by mbleuer** [ch-sagw/sagw#1013]
Header/Main/Footer are lacking a proper labelling.

<img width="300" height="259" alt="Image" src="https://github.com/user-attachments/assets/21ef810d-d833-42fb-bd24-81e08eabbaa2" />
